ereuse_devicehub/templates/ereuse_devicehub/base_site.html

```diff
@@ -220,4 +220,14 @@ <h6>{{ current_user.get_full_name }}</h6>
   </div>
 </footer><!-- End Footer -->
 
+<script>
+  const API_URLS = {
```

Comment:
WARNING: This pull request maybe has a conflict with https://github.com/eReuse/devicehub-teal/pull/219